doc/api/report.md

```diff
@@ -588,6 +588,39 @@ Report version definitions are consistent across LTS releases.
 
 ### Version history
 
+#### Version 5
+
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/56068
+    description: Fix typos in the memory limit units.
+-->
+
+Replace the keys `data_seg_size_kbytes`, `max_memory_size_kbytes` and `virtual_memory_kbytes` with `data_seg_size_bytes`, `max_memory_size_bytes` and `virtual_memory_bytes` respectively in `userLimits` section, as these values are given in bytes.
+
+```json
+{
+  "userLimits": {
+    // Skip some keys ...
+    "data_seg_size_kbytes": { // replaced with data_seg_size_bytes
```

Comment:
I'd suggest using the new keys in the example.

```suggestion
    "data_seg_size_bytes": { // replacing data_seg_size_bytes
```

Reply:
Fixed. Thanks.